example-locale/src/components/footer.tsx

```diff
@@ -0,0 +1,13 @@
+import packageJson from "../../package.json"
+
+export default function Footer() {
+  return (
+    <footer>
+      <p>Example built by</p>
+      <a href="https://alexpatterson.dev" target="_blank" rel="noreferrer">
+        Alex Patterson
+      </a>
```

Comment:
Can we remove this since we're trying to remove all references to any single person in the code base?

Reply:
removed

Reply:
ha, didn't realize the locale stuff, okay now I will remove